app/ts/CreateMarketUI/components/CreateMarket.tsx

```diff
@@ -23,10 +23,15 @@ const isValidDate = (dateStr: string): boolean => {
 	)
 }
 
+const affiliateFeeOptions = [0, 1, 2, 3, 4, 5, 10, 15, 20, 25, 50, 75, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000].map((divisor) => ({
+	id: divisor,
+	name: divisor === 0 ? "0.00%" : `${ (100 / divisor).toFixed(2) }%`
+}))
+
 export const CreateYesNoMarket = ({ maybeAccountAddress }: CreateYesNoMarketProps) => {
 	const endTime = useSignal<string>('')
 	const marketCreatorFee = useSignal<string>('')
-	const affiliateValidator = useSignal<string>('')
+	const affiliateValidator = useSignal<string>('0x0000000000000000000000000000000000000000')
```

Comment:
This is interpreted by the contracts to mean "no affiliate"?

Reply:
yes. I feel its good default